### DISCUSSION TITLE
这个主题真是不错呢, 我每天都有在用

### DISCUSSION THREAD

**Discussion by liangjingkanji** [liangjingkanji/DrakeTyporaTheme#25] [General]
大家有好的风格或者排版可以分享分享

**Comment by arbureva** [comment]
我没有 你有吗 我也想要一些排版方面的资料

**Reply by liangjingkanji** [reply]
现在12个风格不都是我分享的吗?

**Reply by arbureva** [reply]
哦哦哦 我以为你是其他人呢 不好意思哈哈 原来是作者啊 这风格实在是太牛了 非常好看 谢谢大佬

**Comment by callmedd** [comment]
非常非常非常喜欢 Drake 的风格，但是刚刚换上的时候还是有一丢丢不适应，思索了许久，想来提一点点小建议
Drake 拓宽了字符编辑的区域，并且绝大部分都是左对齐，这就导致右侧可能会有非常大的留白，所以有没有可能通过一些线条或者其他的方式让人可以感受到文本可编辑的区域

**Reply by liangjingkanji** [reply]
这大概是你编辑时使用全屏, 你最好能附上截图

**Comment by lutinglt** [comment]
vscode 有个 shades of purple 的主题，感觉紫色配的很好看
